common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -115,6 +114,9 @@ function table(info: SampleReadmeConfiguration) {
     const fileName = info.useTypeScript
       ? relativeSourcePath
       : relativeSourcePath.replace(/\.ts$/, ".js");
+    if (summary && summary.includes("|")) {
```

Comment:
Can we add a test for this? I think you can add the | character to one of the input files and update the expectations here: https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/dev-tool/test/samples/files/README.md

Reply:
add a test https://github.com/Azure/azure-sdk-for-js/pull/29873/commits/c34ff835983057e5a617e0c52c4f4e60ce5fbac9